frame_acceptance:
  sample_count: 10
  accept_count: 5
